packages/ruleset/src/ibm-oas.js

```diff
@@ -34,8 +34,8 @@ module.exports = {
     'openapi-tags': true,
     // Enable with same severity as Spectral
     'operation-description': true,
-    // Turn off - duplicates operation_id_case_convention, operation_id_naming_convention
```

Comment:
Did you check to see if this rule still duplicates these other operation rules?

Reply:
Yes, I did check...  the only thing that the operation-operationId rule does is make sure the operation has an operationId field.  Here's the defn of the rule itself:
```
    'operation-operationId': {
      description: 'Operation must have "operationId".',
      recommended: true,
      type: 'style',
      given: '#OperationObject',
      then: {
        field: 'operationId',
        function: truthy,
      },
    }
```

and btw, the spectral documentation for their rule is a little suspect:
https://meta.stoplight.io/docs/spectral/ZG9jOjExNw-open-api-rules#operation-operationid